libr/bin/format/pe/pe.c

```diff
@@ -1903,12 +1903,12 @@ static void bin_pe_get_certificate (struct PE_ (r_bin_pe_obj_t) * bin) {
 	con = r_pkcs7_parse_cms (data, size);
 	bin->pkcs7 = r_pkcs7_cms_dump (con);
 	if (bin->pkcs7) {
-		ut64 length = strlen(bin->pkcs7);
-		char *c = (char*) malloc(length);
+		ut64 length = strlen (bin->pkcs7);
```

Comment:
actually it handles size_t which is a int64_t on 64 bit machine, and int32_t on 32 bit machine